drop/modules/mae-pipeline/MAE/filterSNVs.sh

```diff
@@ -27,6 +27,7 @@ if [ $vcf_id != 'QC' ]; then sample_flag="-s ${vcf_id}"; fi
 $bcftools view  $vcf_file | \
     grep -vP '^##INFO=' | \
     awk -F'\t' 'BEGIN {OFS = FS} { if($1 ~ /^[^#]/){ $8 = "." }; print $0 }' | \
+    $bcftools norm -m-both | \
     $bcftools view ${sample_flag} -m2 -M2 -v snps -O z -o $tmp
 $bcftools index -t $tmp
```

Comment:
In the `separate_RVC` branch I've updated this to be more robust to multi-sample VCFs. Maybe we can use that version